src/main/steps/application-submitted/content.ts

```diff
@@ -148,12 +162,7 @@ const languages = {
 
 export const generateContent: TranslationFn = content => {
   const { userCase, language, isJointApplication } = content;
-  const currentState = new StateSequence([
-    State.Submitted,
-    State.AwaitingApplicant2Response,
-    State.AwaitingLegalAdvisorReferral,
-    State.FinalOrderComplete,
-  ]).at(content.userCase.state as State);
+  const currentState = currentStateFn(userCase);
   const referenceNumber = userCase.id?.replace(/(\d{4})(\d{4})(\d{4})(\d{4})/, '$1-$2-$3-$4');
```

Comment:
you can use the formattedCaseId function for this

Reply:
does that exist? Can't find it